src/pages/Lend/Filter/FilterRangeSlider.vue

```diff
@@ -0,0 +1,221 @@
+<template>
+	<ais-range-input :attribute="queryAttribute" :min="minimum" :max="maximum">
+		<div slot-scope="{ range, refine }">
+			<div class="header">
+				<span class="title">
+					{{ title }}
+				</span>
+				<span class="range-label">
+					({{ rangeLabel }})
+				</span>
+			</div>
+			<input
+				:id="id"
+				type="range"
+				:min="range.min"
+				:max="range.max"
+				:value="rangeValue"
+				:style="showSliderStyle"
```

Comment:
@foochris did you try just "sliderStyle" here... That's already a reactive property... it should just update without an additional computed layer.

Reply:
@mcstover PR updated with your suggestion.